stdlib/src/builtin/simd.mojo

```diff
@@ -144,6 +144,8 @@ struct SIMD[type: DType, size: Int = simdwidthof[type]()](
         size: The size of the SIMD vector.
     """
 
+    alias Mask = SIMD[DType.bool, size]
```

Comment:
Can you please rename this to `BoolMask`?

Reply:
I still wish it's a top level alias, and since it's not possible right now, shall we hold off for now?

Reply:
The change as is does improve the readability of the implementation, so I think we should have it. Maybe we can just name it `_Mask` or something?

Reply:
I'm gonna amend this change locally so that this becomes a hidden alias.